roles/ipaserver/library/ipaserver_test.py

```diff
@@ -942,7 +943,7 @@ def main():
     else:
         realm_name = options.realm_name.upper()
 
-    argspec = inspect.getargspec(validate_domain_name)
+    argspec = inspect.getfullargspec(validate_domain_name)
```

Comment:
Why is there a change from getargspec to getfullargspec?

Reply:
It shouldn't. Fixed.